xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java

```diff
@@ -1284,7 +1285,19 @@ public void reportLoadStatsToServer() {
     // See more test on LoadReportClientTest.java
   }
 
-  private DiscoveryRpcCall startResourceWatcher(
+  @Test
```

Comment:
These tests shouldn't be written in this way. Tests for XdsClient's operational behaviors (e.g., notify watchers) should be written in the base class, as that's for XdsClient's own logic. See how [other tests](https://github.com/grpc/grpc-java/blob/528ef63c5803f9ead3668d17b3ba99a1c6817ad2/xds/src/test/java/io/grpc/xds/ClientXdsClientTestBase.java#L369) deal with cases not applicable to v2.